packages/types/src/block-kit/blocks.ts

```diff
@@ -44,16 +44,17 @@ export interface Block {
  * {@link https://api.slack.com/reference/block-kit/blocks Blocks reference}.
  */
 export type KnownBlock =
-  | ImageBlock
-  | ContextBlock
   | ActionsBlock
+  | ContextBlock
```

Comment:
Alphabetical 🙏

Reply:
I was not sure where else to add a new block! 😉